dft/src/butterflies.rs

```diff
@@ -76,9 +144,18 @@ impl<F: Field> Butterfly<F> for DitButterfly<F> {
     }
 }
 
-/// Butterfly with twiddle factor 1 (works in either DIT or DIF).
+/// Butterfly with no twiddle factor (twiddle = 1).
+///
+/// This is used when no root-of-unity scaling is needed.
+/// It works for either DIT or DIF, and is often used at
+/// the final or base level of a transform tree.
+///
+/// Computes:
+///   output_1 = x1 + x2
+///   output_2 = x1 - x2
```

Comment:
Should either surround by \```text \``` or:
```suggestion
/// Computes:
///   - `output_1 = x1 + x2`
///   - `output_2 = x1 - x2`
```
Also just a minor nit but can you make the formatting of the doc comments for `DifButterfly, DitButterfly, TwiddleFreeButterfly` be identical?

E.g. either use:
```
computes:
 - output_1 = ...
 - output_2 = ...
```
or
```
     output_1 = ...
     output_2 = ...
```
for all of them.